loadtesting/k6/tests/utils/ethClient.js

```diff
@@ -12,11 +12,16 @@ export function ethClient(privateKey) {
 }
 
 export function sendNeon(client, from, to, amount, gas, gasPrice, nonce) {
-    const value = amount;
-    return sendTokens(client, from, to, value, gas, gasPrice, nonce)
+    return sendTokens(client, from, to, amount, null, gas, gasPrice, nonce)
 }
 
-export function sendTokens(client, from, to, value, gas, gasPrice, nonce) {
+export function sendErc20ViaTransferFunction(client, erc20, owner, abi, signerAddress, receiverAddress, amount) {
+    const input = erc20.fillInput(abi, "transfer");
+    const nonce = client.getNonce(signerAddress);
+    return sendTokens(client, owner, receiverAddress, amount, input, null, null, nonce)
```

Comment:
looks strange 
for erc20 "amount" should be part of input field (or data field) 
and this ""value": value," should be 0 all the time

Reply:
fixed: full changes of input calculations are in xk6 plugin